src/Storyblok/BaseClient.php

```diff
@@ -268,7 +285,7 @@ protected function getResponseExceptionMessage(\GuzzleHttp\Message\Response $res
     /**
      * Gets the json response body
      *
-     * @return array
+     * @return array|stdClass
```

Comment:
this is correct (looking at ResponseHandler), it is even: array|stdClass|string (when body cannot be decoded as json)

but nonetheless, i think we should change: 

```
public $responseBody;
``` 
in Line 26 as well to have a type?